frontend/model/state.js

```diff
@@ -600,11 +596,17 @@ const actions = {
   },
   // persisting the state
   async saveSettings (
-    { state }: {state: Object}
+    { commit, state }: { state: Object}
```

Comment:
Can you remove this `commit` from here just to be safe, so that we don't give the impression that it's safe to use in this function? (Maybe even add a comment to that effect...)

Reply:
Sure. Good catch.